CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- [DesignToolbox] Update cards backgrounds token (Orange-OpenSource/ouds-ios#559)
```

Comment:
The related issue is n°696 and not n°559 (which is here an old pull request).
Use instead:

```text
- [DesignToolbox] Update cards background color token (Orange-OpenSource/ouds-ios#695)
```